packages/providers/providers.yaml

```
@@ -12602,7 +12602,36 @@ todoist:
     proxy:
         base_url: https://api.todoist.com
     docs: https://nango.dev/docs/integrations/all/todoist
-
+totango:
+    display_name: Totango
+    categories:
+        - dev-tools
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.totango.com/api/v3
+        headers:
+            authorization: app-token ${appToken}
+            service_id: ${serviceId}
+        verification:
+            method: GET
+            endpoints:
+                - /v0/me
+    docs: https://nango.dev/docs/integrations/all/totango
+    docs_connect: https://nango.dev/docs/integrations/all/totango/connect
+    credentials:
+        appToken:
+            type: string
+            title: App Token
+            description: The App Token for your Totango account
+            doc_section: '#instructions'
+            example: 9164000970ca2f15c999047a0e2b7e03127885be41198616a6975af08b28239b
+        serviceId:
+            type: string
+            title: Service ID
+            description: The Service ID for your Totango account (use only the middle number, e.g., if your service ID is SP-275419-01, use 275419)
+            doc_section: '#instructions'
```

```suggestion
            doc_section: '#step-1-finding-your-app-token-and-service-id'
```
Same as above. you can also `npm run test:providers` to test the providers configuration.